src/main/java/dk/cs/aau/huppaal/controllers/EdgeController.java

```diff
@@ -11,10 +11,8 @@
 import dk.cs.aau.huppaal.presentations.NailPresentation;
 import dk.cs.aau.huppaal.utility.UndoRedoStack;
 import dk.cs.aau.huppaal.utility.colors.Color;
-import dk.cs.aau.huppaal.utility.helpers.BindingHelper;
-import dk.cs.aau.huppaal.utility.helpers.Circular;
-import dk.cs.aau.huppaal.utility.helpers.ItemDragHelper;
-import dk.cs.aau.huppaal.utility.helpers.SelectHelper;
+import dk.cs.aau.huppaal.utility.helpers.*;
```

Comment:
our static analysis engine detects that your pull request includes change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.